builtins/safe-default-configuration.txt

```diff
@@ -243,3 +243,200 @@ math semantics
 - mathsize
 - scriptlevel
 
+
+// SVG 2
+// https://svgwg.org/svg2-draft/
+// as referenced here: https://html.spec.whatwg.org/#refsSVG
+
+// Document structure, https://svgwg.org/svg2-draft/struct.html
+svg svg
+- viewBox
+- preserveAspectRatio
+- transform
+svg g
+svg defs
+svg symbol
+- preserveAspectRatio
+- viewBox
+- refX
+- refY
+// Purposely omitted: svg use.
+svg switch
+svg title
+svg desc
+svg metadata
+
+// Paths, https://svgwg.org/svg2-draft/paths.html
+svg path
+- pathLength
+- d
+
+// Shapes, https://svgwg.org/svg2-draft/shapes.html
+svg rect
+- pathLength
+- x
+- y
+- width
+- height
+- rx
+- ry
+svg circle
+- pathLength
+- cx
+- cy
+- r
+svg ellipse
+- pathLength
+- cx
+- cy
+- rx
+- ry
+svg line
+- pathLength
+- x1
+- y1
+- x2
+- y2
+svg polyline
+- pathLength
+- points
+svg polygon
+- pathLength
+- points
+// Text, https://svgwg.org/svg2-draft/text.html
+svg text
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg tspan
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg textPath
+- lengthAdjust
+- textLength
+- path
+- href
+- startOffset
+- method
+- spacing
+- side
+
+// Embedded Content, https://svgwg.org/svg2-draft/embedded.html
+svg image
+- preserveAspectRatio
+- href
+- crossorigin
+- x
+- y
+- width
+- height
+svg foreignObject
+- x
+- y
+- width
+- height
+// Painting, https://svgwg.org/svg2-draft/painting.html
+svg marker
+- viewBox
+- preserveAspectRatio
+- refX
+- refY
+- markerUnits
+- markerWidth
+- markerHeight
+- orient
+
+// Purposely omitted: svg script
+
+// Common attributes: https://svgwg.org/svg2-draft/struct.html#CommonAttributes
+[SVG Global]
+- id
+- lang
+- xml space
+- tabindex
```

Comment:
Actual a bit similar to autofocus, I think this could change the global tab order, which might make it easy to trick keyboard users into activating something they didn't mean to.